app/views/how-tos/github/download-existing-prototype.html

```diff
@@ -0,0 +1,61 @@
+{% extends "layout.html" %} {% block pageTitle %} Download an existing prototype
+from GitHub - NHS prototype kit {% endblock %} {% block beforeContent %} {{
+breadcrumb({ items: [ { href: "/", text: "Home" }, { href: "/how-tos", text:
+"Guides" }, { href: "/how-tos/github/index", text: "Store code online with
+GitHub and GitHub desktop" } ] }) }} {% endblock %} {% block content %}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-three-quarters">
+    <h1 class="nhsuk-heading-xl">Download an existing prototype from GitHub</h1>
+
+    <p>
+      To store your new prototype in GitHub, you need to
+      <a href="index">set up GitHub Desktop</a>. If you work for NHS England,
```

Comment:
Doubt we want to link to a url with `index`?

Reply:
changed to absolute url to start of section